### DISCUSSION TITLE
Order of cameras on main screen

### DISCUSSION THREAD

**Discussion by RegProctor** [Motion-Project/motion#1670] [Q&A] [answered]
Is there a way on the main screen that shows all the cameras at once to specify the order that the cameras appear? I didn't find that bit of information anywhere I looked in the documentation.

**Comment by hormigaclub** [comment]
camera_id ?

El vie, 5 may 2023 a la(s) 04:07, Reg ***@***.***) escribió:

> Is there a way on the main screen that shows all the cameras at once to
> specify the order that the cameras appear? I didn't find that bit of
> information anywhere I looked in the documentation.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Motion-Project/motion/discussions/1670>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAR3YQFVQIRVD6PAA6OKWT3XESRKDANCNFSM6AAAAAAXWXI7WM>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Gustavo Javier Ovalle
Ituzaingo 1365
S3070BPW - San Cristóbal
Santa Fe
e-mail: ***@***.***

**Comment by Mr-Dave** [comment, marked as answer]
The sequence they are listed in the `motion.conf` file.  If you use the `camera_dir` option, it is the order that motion reads the directory which is not controllable.

**Reply by RegProctor** [reply]
In that case it's not controllable for me.  You would think there would be a sort by file name or something but I guess I can change to other method so I have control over the order.
